README.markdown

```diff
@@ -192,6 +192,38 @@ load-nvmrc() {
 }
 add-zsh-hook chpwd load-nvmrc
 ```
+##### Automatic version switching for `.bashrc` with fallback to `nvm use default`
+
+Put this into your `$HOME/.bashrc` to enable automatic version swithing whenever you enter a directory than contains an
+`.node-version` file with a string telling nvm which node to `use`:
```

Comment:
`nvm` supports `.nvmrc` - `.node-version` is part of avn but isn't actually a community thing.